Database/init-constraints.sql

```diff
@@ -1,17 +1,17 @@
 CREATE OR REPLACE FUNCTION
-common.check_fk_exists(object_id INT, object_type_id INT)
+common.check_fk_exists(object_id INT, object_type common.object_type)
 RETURNS BOOLEAN AS $$
 BEGIN
-	IF object_type_id = 1 THEN
+	IF object_type = 'Event' THEN
 	RETURN EXISTS (SELECT 1 FROM users.user WHERE "id" = object_id);
-	ELSIF object_type_id = 2 THEN RETURN EXISTS (SELECT 1 FROM events.event WHERE "id" = object_id);
+	ELSIF object_type = 'User' THEN RETURN EXISTS (SELECT 1 FROM events.event WHERE "id" = object_id);
 	ELSE RETURN FALSE;
 	END IF;
```

Comment:
Should't it be that if 'Event' then get from events.event, if 'User' then from users.user